src/index.ts

```diff
@@ -1,6 +1,15 @@
 // re-export
-export * from './bootstrap';
-export * from './launch';
+
+import { bootstrap } from './bootstrap';
+import { launch } from './launch';
+import { getWallet } from './wallets/wallets';
+
+const defaultObject = { bootstrap, launch, getWallet };
+export default defaultObject;
```

Comment:
is this `defautlObject` export being used anywhere?

Reply:
correct!